openapi/components/schemas/AmlCompoundConfidence.yaml

```
@@ -1,38 +1,28 @@
 type: object
 properties:
-  cityAndRegionMatch:
+  addressMatch:
     type: object
-    description: Exact match of the customer's city and region of residence is found.
+    description: A match for the customer's city and/or region is found.
```

suggestion: to avoid using `and/or` 
```suggestion
    description: A match for the customer's city or region, or both, is found.
```